<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QFrame" name="contentFrame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="addToDoList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QFrame" name="addRight">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color: #ff3333;
	color: #fff;
	font-size: 12px;
	font-weight: bold;
	border-radius: 8px;
	padding: 10px;
}

QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::pressed
{
	background-color: #ff4747;

}

QPushButton::hover
{
	background-color: #8E8E8E;

}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="addNewBtn">
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	background-color: #ff3333;
	color: #fff;
	font-size: 12px;
	font-weight: bold;
	border-radius: 8px;
	padding: 10px;
}

QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::pressed
{
	background-color: #ff4747;

}

QPushButton::hover
{
	background-color: #8E8E8E;

}</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Sort By Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Sort By Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="prirSortButton">
               <property name="text">
                <string>Sort By Priority</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="addLeft">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="newTaskLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #FFFFFF;
	color: #D4D4D4;
	font-weight: bold;
	border: solid;
	border-width: 2px;
	border-radius: 4px;
	border-color: #E5E5E5;
	padding: 3px;

}</string>
               </property>
               <property name="text">
                <string>e.g Buy gift tomorrow at 6pm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="newTaskDesLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>80</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #FFFFFF;
	color: #D4D4D4;
	font-weight: bold;
	border: solid;
	border-width: 2px;
	border-radius: 4px;
	border-color: #E5E5E5;
	padding: 3px;

}</string>
               </property>
               <property name="text">
                <string>Description</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Low</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Medium</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>High</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QCalendarWidget" name="calendarWidget">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>180</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="gridVisible">
                <bool>false</bool>
               </property>
               <property name="horizontalHeaderFormat">
                <enum>QCalendarWidget::SingleLetterDayNames</enum>
               </property>
               <property name="verticalHeaderFormat">
                <enum>QCalendarWidget::NoVerticalHeader</enum>
               </property>
               <property name="navigationBarVisible">
                <bool>true</bool>
               </property>
               <property name="dateEditEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="titleBarFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame 
{
	background-color: #DC4C3F;
	color: #FFFFFF;
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel 
{

	font-size: 14px;
	font-weight: bold;
}	
</string>
         </property>
         <property name="text">
          <string>todoApp</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>336</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="plus_image">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="profile_image">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QScrollArea" name="allNewTasks">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="allNewTasksContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>554</width>
         <height>72</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/*-----QLineEdit-----*/
QWidget
{
	background-color: #FFFFFF;
	color: #535452;
	font-weight: bold;
	border: solid;
	border-width: 2px;
	border-radius: 4px;
	border-color: #E5E5E5;
	padding: 3px;

}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>45</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame 
{
	background-color: #F0EFEF;
}

QPushButton
{
	background-color: #ff3333;
	color: #fff;
	font-size: 12px;
	font-weight: bold;
	border-radius: 8px;
	padding: 10px;
}

QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::pressed
{
	background-color: #ff4747;

}

QPushButton::hover
{
	background-color: #8E8E8E;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="listHeaderFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLineEdit" name="newListLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #FFFFFF;
	color: #D4D4D4;
	font-weight: bold;
	border: solid;
	border-width: 2px;
	border-radius: 4px;
	border-color: #E5E5E5;
	padding: 3px;

}
</string>
            </property>
            <property name="text">
             <string>eg. grocery shopping</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addNewListBtn">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>New List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*-----QLineEdit-----*/
QLabel
{
	background-color: #FFFFFF;
	color: #D4D4D4;
	font-weight: bold;
	border: solid;
	border-width: 2px;
	border-radius: 4px;
	border-color: #E5E5E5;
	padding: 3px;

}</string>
         </property>
         <property name="text">
          <string>My List:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="listFrame">
         <property name="styleSheet">
          <string notr="true">QFrame
{
	background-color: #F0EFEF;
	color: #585757;
	font-weight: bold;
	border: solid;
	
	border-radius: 4px;

	padding: 3px;

}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuMark_Due_Date_As">
     <property name="title">
      <string>Mark Due Date As</string>
     </property>
     <addaction name="actionToday"/>
     <addaction name="actionTomorrow"/>
     <addaction name="actionThis_Weekend"/>
     <addaction name="separator"/>
     <addaction name="actionNone"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="menuMark_Due_Date_As"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New Reminder</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>New List</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>New Group</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionToday">
   <property name="text">
    <string>Today</string>
   </property>
  </action>
  <action name="actionTomorrow">
   <property name="text">
    <string>Tomorrow</string>
   </property>
  </action>
  <action name="actionThis_Weekend">
   <property name="text">
    <string>This Weekend</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
